### DISCUSSION TITLE
스타일시트 import를 styles로 통일하기

### DISCUSSION THREAD

**Discussion by JiinSeok** [team-final/albaform#186] [General]
어떤가요?
파일 안에 많은 style이 있으니까 `styles`

여러 컴포넌트에서 사용되는 스타일 파일은 `<사용되는 곳들을 총칭할 수 있는 주제어>Styles`

**Comment by JuhyeokC** [comment]
현재 컴포넌트를 위한, 같은 디렉토리에서 가져오는 `'./Component.module.scss` 는 `styles`
다른 컴포넌트 디렉토리에서 가져오는 scss 는 `해당 컴포넌트명Styles`

정도면 될 것 같습니다.
